source/includes/gho/_api.md

```diff
@@ -0,0 +1,305 @@
+
+# Input Objects
+## DateFilter
+Specify a range of dates using after (exclusive >), before (exclusive <), or on (exact match)
+
+Argument | Type | Description
+--------- | ----------- | -----------
+after | [Date](#date) | 
+before | [Date](#date) | 
+on | [Date](#date) | 
+
+## DateTimeFilter
+Specify a range of datetimes using after (exclusive >), before (exclusive <), or on (exact match)
+
+Argument | Type | Description
+--------- | ----------- | -----------
+after | [DateTime](#datetime) | 
+before | [DateTime](#datetime) | 
+on | [DateTime](#datetime) | 
+
+## UpdateCustomFieldValue
+Used to update an employee's Custom Field Value
+
+Argument | Type | Description
+--------- | ----------- | -----------
+permanent_field_id | [String](#string) | 
+value | [Value](#value) | 
+
+## UpdateEmployee
+Used to update an Employee.
+
+Argument | Type | Description
+--------- | ----------- | -----------
+about | [String](#string) | 
+date_of_birth | [Date](#date) | 
+department | [ID](#id) | 
+email | [String](#string) | 
+employment_status | [EmploymentStatus](#employmentstatus) | 
+first_name | [String](#string) | 
+hr_manager | [ID](#id) | 
+last_name | [String](#string) | 
+location | [ID](#id) | 
+manager | [ID](#id) | 
+middle_name | [String](#string) | 
+personal_email | [String](#string) | 
+phone_number | [String](#string) | 
+preferred_first_name | [String](#string) | 
+preferred_last_name | [String](#string) | 
+profile_image | [URL](#url) | 
+start_date | [Date](#date) | 
+suffix | [String](#string) | 
+title | [String](#string) | 
+work_country_code | [String](#string) | 
+custom_field_values | [UpdateCustomFieldValue](#updatecustomfieldvalue) | 
+
+## WithCustomFieldValue
+Limit employees to those that contain at least one of the specified CustomFieldValues that have been updated in the provided time range
+
+Argument | Type | Description
+--------- | ----------- | -----------
+permanent_field_ids | [String](#string) | 
+updated | [DateTimeFilter](#datetimefilter) | 
+
+# Scalars
+## Boolean
+Represents `true` or `false` values.
+
+## Date
+Representation of a date in YYYY-MM-DD format
+
+## DateTime
+Representation of datetime in RFC3339
+
+## ID
+Represents a unique identifier that is Base64 obfuscated. It is often used to refetch an object or as key for a cache. The ID type appears in a JSON response as a String; however, it is not intended to be human-readable. When expected as an input type, any string (such as `"VXNlci0xMA=="`) or integer (such as `4`) input value will be accepted as an ID.
+
+## Int
+Represents non-fractional signed whole numeric values. Int can represent values between -(2^31) and 2^31 - 1.
+
+## String
+Represents textual data as UTF-8 character sequences. This type is most often used by GraphQL to represent free-form human-readable text.
+
+## URL
+A URL-formatted String
+
+## Value
+The actual value of a Custom Field Value. Can be of an arbitrary type (string, int, object, etc.)
+
+# Enums
```

Comment:
Might be worth mentioning that `Enums` are unquoted in user input and quoted in our output